content/6_Advanced/XOR_Basis.mdx

```diff
@@ -25,8 +24,254 @@ frequency: 1
 	>
 		used at USACO Camp
 	</Resource>
+	<Resource
+		source="freakin23"
```

Comment:
Hoffman + Kunze?

Reply:
also I'm not really sure if it's necessary to read an entire textbook on linear algebra to understand xor basis

Reply:
> Hoffman + Kunze?

ok will change this

Reply:
> also I'm not really sure if it's necessary to read an entire textbook on linear algebra to understand xor basis

It depends on the user. I already mentioned the important terms... if they feel some doubt then they can always refer to the book for more clarification